utils/RunONNXModel.py

```diff
@@ -74,10 +74,19 @@ def valid_onnx_input(fname):
 parser.add_argument('--verify',
                     choices=['onnxruntime', 'ref'],
                     help="Verify the output by using onnxruntime or reference"
-                    " inputs/outputs. By default, no verification")
+                    " inputs/outputs. By default, no verification. When being"
+                    " enabled, --verify-with-softmax or --verify-every-value"
+                    " must be used to specify verification mode")
```

Comment:
Explain more about `-verify` that need to specify verification mode via `--verify-with-softmax` or `--verify-every-value`